install.sh

```diff
@@ -7,43 +7,22 @@ echo "# This script installs the dependencies for emirge. #"
 echo "#####################################################"
 echo
 
-if [[ ! -f meshmode/setup.py ]]; then
-	echo "ERROR: incomplete git clone. Please run:"
-	echo "  git submodule init && git submodule update"
-	echo "to fetch emirge's submodules."
-	exit 1
-fi
+export MY_CONDA_DIR=$HOME/miniforge3
 
-myos=$(uname)
-[[ $myos == "Darwin" ]] && myos="MacOSX"
+./install-conda.sh
 
-myarch=$(uname -m)
+export PATH=$MY_CONDA_DIR/bin:$PATH
```

Comment:
Tempted to say this should go in `install-conda.sh`.

Reply:
You can't export variables from a sub-script to its calling script.